dev/buildtool/inspection_commands.py

```diff
@@ -1284,7 +1284,11 @@ def audit_service(service, versions):
         for _, buildnums in commits.items():
           for buildnum, info_list in buildnums.items():
             version_buildnum = '%s-%s' % (version, buildnum)
-            jar_ok = self.audit_jar(service, version_buildnum, info_list)
+            if service in ['monitoring-daemon', 'monitoring-third-party']:
+              # Uses debians, but not jars
```

Comment:
I'd add a "so jars_ok is trivially true" to explicitly describe what we're doing here.